### DISCUSSION TITLE
Importing media from thumb drive or computer

### DISCUSSION THREAD

**Discussion by DPBradley58** [ChurchApps/ChurchAppsSupport#334] [FreeShow] [answered]
I don't see my media files in the media bin/drawer when I drag them in. Am I doing it the wrong way?

**Comment by jzongker** [comment, marked as answer]
You can drag/drop media into the list of slides for a show, or into the project pane to create a new show from it, but not into the media area itself.  You'll need to click the "+ New Folder" button instead to add whichever folder contains your files.
